pathToCheckpoint: "./FastSAM/Models/FastSAM-x.pt"
#pathToCheckpoint: "sam_b.pt"
device: "cpu"
conf: 0.5
iou: 0.9